webapp/src/Helpers/OrdinalItem.php

```diff
@@ -13,16 +13,12 @@
  */
 class OrdinalItem
 {
-    /** @Serializer\SerializedName("ordinal") */
-    protected int $ordinal;
-
-    /** @Serializer\Inline() */
-    protected object $item;
-
-    public function __construct(int $ordinal, object $item)
-    {
-        $this->ordinal = $ordinal;
-        $this->item    = $item;
+    public function __construct(
+        /** @Serializer\SerializedName("ordinal") */
```

Comment:
Do we check if moving this here keeps the stuff in the API doc correct?

Reply:
I did and it does. It’s not only the doc, it’s the actual output

Reply:
I didn't think of that, you're right.